service/client/client.js

```diff
@@ -110,8 +110,8 @@ export default class Client {
         return `${this.url}${this.urlVersion}/users`;
     }
 
-    getFilesRoute(teamId) {
-        return `${this.url}${this.urlVersion}/teams/${teamId}/files`;
+    getFilesRoute() {
```

Comment:
I think you have to add another one as there is the getFilesRoute that needs the team in order to upload files